### DISCUSSION TITLE
New to Gsoc

### DISCUSSION THREAD

**Discussion by BoxuLiu** [cncf/mentoring#907] [Google Summer of Code 2023]
Hello everyone, It's my first time contributing to an open-source program. I find these ideas really interesting:
              1. Improving Observability in Knative Eventing
              2. Dataplane migration for Apache Kafka communications: From Vert.x to Project Loom

I've been learning Knative in recent days and preparing my project proposal. I'm wondering where can I get feedback on my proposal. should I send it to the mentor personally or just post it here?

**Comment by aliok** [comment]
hey @BoxuLiu 

We encourage you to join community channels for the projects you're interested in and reach out to community members and mentors for questions and also for asking a review of your proposal.

**Comment by ptiwari6239** [comment]
Hye , I am new to open source community ,can you guys please guide or point out some resources so that I can start contributing to this community, although I worked with docker , kubernetes and jenkins is it enough?

**Reply by aliok** [reply]
Hey @ptiwari6239 👋 

It is very nice to see that you're interested in contributing to open source and CNCF.

We have a very recent article that you might want to take a look. It includes all kind of information an open-source-beginner might need: 
- Title: "Do you want to start contributing to open source and need help figuring out where to begin?"
- Link: https://contribute.cncf.io/contributors/getting-started/

**Comment by kushalShukla-web** [comment]
@aliok is there any gitter channel for this organization ??

**Reply by aliok** [reply]
hey @kushalShukla-web 

We don't use Gitter officially. There might be communities using it, I don't know.

We're on Slack though: https://communityinviter.com/apps/cloud-native/cncf
